.travis.yml

```diff
@@ -2,7 +2,8 @@ language: scala
 jdk:
   - oraclejdk8
```

Comment:
I reverted the commit that changed java-it-tests so it supported jdk6.   Also updated build.gradle to remove source/target compatibility and project root level.  Travis will now build with jdk8, run the java-it-tests at jdk8 and the rest of the tests at jdk6.